clean-blog/static/css/bootstrap.cyborg.css

```diff
@@ -0,0 +1,6817 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:400,700&display=swap");
```

Comment:
Es wäre schön wenn google Fonts hier raus fliegen könnte.

(https://google-webfonts-helper.herokuapp.com/fonts)[https://google-webfonts-helper.herokuapp.com/fonts] ist da sehr hilfreich bei

Reply:
ah, da habe ich nicht dran gedacht

wobei: das aktuelle Theme hat auch schon diverse Fonts von externen CDNs
Ich habe kein Problem damit das rauszuschmeißen, denke aber, dass das für den PR out of scope ist.

Reply:
Agreed. Ich hab einen issue dazu aufgemacht: #5

Reply:
Hab das mal als Issue #6 aufgemacht